src/event/event.rs

```diff
@@ -62,59 +62,51 @@ impl Event {
         sys::event::is_error(&self.inner)
     }
 
-    /// Returns true if the event contains HUP readiness.
+    /// Returns true if the event contains read close readiness.
```

Comment:
We'll need to expand this documentation to explain when "read close" readiness can be expected, can be a follow up pr. Same for write docs of course.

Reply:
Docs have been updated to included the events when these methods _should_ return `true`